internal/serve/httphandler/payments_handler_test.go

```diff
@@ -42,9 +42,9 @@ func TestSubscribeAddress(t *testing.T) {
 		require.NoError(t, err)
 	}
 
-	t.Run("success", func(t *testing.T) {
+	t.Run("success_happy_path", func(t *testing.T) {
```

Comment:
Just out of curiosity, why are we naming with camel case the test name?

Reply:
I decided to start naming it with snake case as opposed to with spaces because it makes it easier to search for a failing test based on the `go test` output (which replaces whitespaces with `_`).
e.g.: ` --- PASS: TestUnsubscribeAddress/success_happy_path (0.00s)`

But I'm also open to suggestions here, do you think camelCase would be more appropriate?

Reply:
Got it! Thank you, It may help find the test when it fails.